packages/client/hmi-client/src/workflow/ops/subset-data/tera-subset-data.vue

```diff
@@ -0,0 +1,396 @@
+}
+
+const dataset = ref<Dataset | null>(null);
+const preview = ref<string | undefined>(undefined);
+
+const subset = ref<Dataset | null>(null);
+const isSubsetLoading = ref(false);
+
+const latitudeStart = ref(props.node.state.latitudeStart);
+const latitudeEnd = ref(props.node.state.latitudeEnd);
+const longitudeStart = ref(props.node.state.longitudeStart);
+const longitudeEnd = ref(props.node.state.longitudeEnd);
+
+const isSpatialSkipping = ref(props.node.state.isSpatialSkipping);
+const spatialSkipping = ref(props.node.state.spatialSkipping);
+
+const isTimeSkipping = ref(props.node.state.isTimeSkipping);
+const timeSkipping = ref(props.node.state.timeSkipping);
+
+const fromDate = ref(new Date(props.node.state.fromDate));
+const toDate = ref(new Date(props.node.state.toDate));
+
+const selectedOutputId = ref(props.node.active ?? '');
+
+const showSaveDatasetModal = ref(false);
+const newDatasetName = ref('');
+
+const outputs = computed(() => {
+	if (!isEmpty(props.node.outputs)) {
+		return [
+			{
+				label: 'Select output to display in operator',
+				items: props.node.outputs
+			}
+		];
+	}
+	return [];
+});
+
+const onSelection = async (id: string) => {
+	emit('select-output', id);
+};
+
+function updateState() {
+	const state = cloneDeep(props.node.state);
+	state.datasetId = dataset.value?.id ?? null;
+	state.latitudeStart = latitudeStart.value;
+	state.latitudeEnd = latitudeEnd.value;
+	state.longitudeStart = longitudeStart.value;
+	state.longitudeEnd = longitudeEnd.value;
+	state.isSpatialSkipping = isSpatialSkipping.value;
+	state.spatialSkipping = spatialSkipping.value;
+	state.isTimeSkipping = isTimeSkipping.value;
+	state.timeSkipping = timeSkipping.value;
+	state.fromDate = fromDate.value;
+	state.toDate = toDate.value;
+	emit('update-state', state);
+}
+
+async function saveSubsetAsNewDataset() {
+	const projectId = useProjects().activeProject.value?.id;
+	if (subset.value && projectId) {
+		const subsetToSave = cloneDeep(subset.value);
+		subsetToSave.name = newDatasetName.value;
+		const newDataset = await createDataset(subsetToSave);
+
+		if (newDataset) {
+			await useProjects().addAsset(AssetType.Dataset, newDataset.id, projectId);
+			logger.info(`New dataset saved as ${newDataset.name}`);
+			showSaveDatasetModal.value = false;
+			newDatasetName.value = subset.value?.name ?? '';
+		} else {
+			logger.error('Failed to save subset as new dataset');
+		}
+	} else {
+		logger.error('Subset not found');
+	}
+}
+
+async function run() {
+	if (dataset.value?.esgfId && dataset.value?.id) {
+		updateState();
+		isSubsetLoading.value = true;
+
+		const subsetId = await getClimateSubsetId(
+			dataset.value.esgfId,
+			dataset.value.id,
+			`${longitudeStart.value},${longitudeEnd.value},${latitudeStart.value},${latitudeEnd.value}`,
+			`${fromDate.value.toISOString()},${toDate.value.toISOString()}`
+			// isSpatialSkipping.value ? spatialSkipping.value ?? undefined : undefined // Not sure if its this or timeSkipping
```

Comment:
Do we have an answer now?

Reply:
After asking Satchel it seems like the UI doesn't match with how the climate data service handles this.

If you look at where it says `thin-factor` a bit below here https://github.com/DARPA-ASKEM/climate-data?tab=readme-ov-file#preview

It only accepts one `n` value and then you can exclude certain fields to be effected using the `thin-fields` param (this param isn't in our call yet).

Maybe for this pass we can ignore this?

CC @dvince2 @bigglesandginger